packages/core/core/src/UncommittedAsset.js

```diff
@@ -62,7 +60,7 @@ export default class UncommittedAsset {
   options: ParcelOptions;
   content: ?(Blob | Promise<Buffer>);
   mapBuffer: ?Buffer;
-  sourcesContent: ?SourcesContentDictionary;
```

Comment:
Why did this change to just one source content? Compiled modules can have multiple sources?

Reply:
Extends should take care of setting source content when there is a previous source map right?

Reply:
In case of babel it's part of the ast so no extends should be called and therefore it should use sourcescontent

Reply:
Kind of, the only issue here is that the map will be erased whenever `commit` is called in `UncommittedAsset.js` so those source contents get lost pretty quickly if we ever re-use the AST... which is maybe no longer obvious after the JS Transformer rewrite but still a bug...

Reply:
This was how I originally fixed this issue, before I saw this line: https://github.com/parcel-bundler/parcel/blob/v2/packages/core/core/src/UncommittedAsset.js#L102

Reply:
Oh yeah I guess it would only be an issue if the packager uses an AST but that's not the case anymore. Kinda wondering if we should just remove support for that.

Reply:
Sounds like a good idea to remove support for it. Would probably cause issues at some point otherwise